shared/tracker/remote-proxy.desktop.js

```diff
@@ -5,33 +5,89 @@
 // import * as I from 'immutable'
 import * as React from 'react'
 import * as Constants from '../constants/tracker'
+import {parsePublicAdmins} from '../util/teams'
+
 import SyncAvatarProps from '../desktop/remote/sync-avatar-props.desktop'
 import SyncProps from '../desktop/remote/sync-props.desktop'
 import SyncBrowserWindow from '../desktop/remote/sync-browser-window.desktop'
-import {connect, type TypedState, compose, renderNothing} from '../util/container'
+import {connect, type TypedState, compose, renderNothing, withState} from '../util/container'
 
 const MAX_TRACKERS = 5
 const windowOpts = {height: 470, width: 320}
 
-const trackerMapStateToProps = (state: TypedState, {name}) => {
+const trackerMapStateToProps = (state: TypedState, {name, showTeam}) => {
+  const teamname = showTeam && showTeam.fqName
+  const myUsername = state.config.username
+  // If the current user's in the list of public admins, pull them out to the front.
+  let publicAdmins = []
+  let publicAdminsOthers = 0
+  if (showTeam) {
+    ;({publicAdmins, publicAdminsOthers} = parsePublicAdmins(showTeam.publicAdmins, myUsername))
+  }
+
   return {
     _trackerState: state.tracker.userTrackers[name] || state.tracker.nonUserTrackers[name],
+    description: showTeam && showTeam.description,
+    following: state.config.following.toObject(),
```

Comment:
this should happen in merge and not mapStateToProps

Reply:
Thanks, done.